elastic/kibana
# [ResponseOps][Rules][Alerts] Link stack rule details page refresh button to alerts table

## Summary

Forwards the reload request from the Stack Management rule detail page `🔄 Refresh` button to the alerts table.

## To verify

1. Create a Stack rule that fires alerts
2. Visit that rule's detail page
3. Check that the `🔄 Refresh` button correctly refreshes the alerts table (i.e. by changing the rule params and checking that the alerts update accordingly)

## References

Fixes #188405, #183155
